app/src/main/java/ru/thstdio/aa2020/cache/workmanager/CinemaWorkManager.kt

```diff
@@ -0,0 +1,35 @@
+package ru.thstdio.aa2020.cache.workmanager
+
+import android.content.Context
+import android.util.Log
+import androidx.work.Constraints
+import androidx.work.NetworkType
+import androidx.work.PeriodicWorkRequest
+import androidx.work.WorkManager
+import java.util.concurrent.TimeUnit
+
+const val WORK_TAG = "CinemaWorkManager_Tag"
```

Comment:
why u need this class at all?
move code to CinemaWorker companion
or make this class as object